src/parser/SparqlParser.cpp

```diff
@@ -63,7 +63,7 @@ void SparqlParser::parsePrologue(string str, ParsedQuery& query) {
 
 // _____________________________________________________________________________
 void SparqlParser::addPrefix(const string& str, ParsedQuery& query) {
-  auto parts = ad_utility::split(ad_utility::strip(str, ' '), ' ');
+  auto parts = ad_utility::splitWs(ad_utility::strip(str, ' '));
```

Comment:
Since splitWs removes the whitespace from the splits shouldn't the strip() be redudant?